src/tests/ftest/pool/attribute.py

```diff
@@ -30,123 +25,152 @@ def cb_func(event):
     GLOB_SIGNAL.set()
 
 
-def verify_list_attr(indata, size, buff):
-    """
-    verify the length of the Attribute names
-    """
-    # length of all the attribute names
-    aggregate_len = sum(len(attr_name) for attr_name in indata.keys()) + 1
-    # there is a space between each name, so account for that
-    aggregate_len += len(indata.keys())-1
-
-    if aggregate_len != size:
-        raise DaosTestError("FAIL: Size is not matching for Names in list"
-                            "attr, Expected len={0} and received len = {1}"
-                            .format(aggregate_len, size))
-    # verify the Attributes names in list_attr retrieve
-    for key in indata.keys():
-        if key not in buff:
-            raise DaosTestError("FAIL: Name does not match after list attr,"
-                                " Expected buf={0} and received buf = {1}"
-                                .format(key, buff))
-
-
-def verify_get_attr(indata, outdata):
-    """
-    verify the Attributes value after get_attr
-    """
-    for attr, value in indata.iteritems():
-        if value != outdata[attr]:
-            raise DaosTestError("FAIL: Value does not match after get attr,"
-                                " Expected val={0} and received val = {1}"
-                                .format(value, outdata[attr]))
+class PoolAttributeTest(TestWithServers):
+    """Pool Attribute test Class.
 
+    Test class Description:
+        Tests DAOS pool attribute get/set/list.
 
-class PoolAttributeTest(TestWithServers):
-    """
-    Test class Description: Tests DAOS pool attribute get/set/list.
     :avocado: recursive
     """
 
-    def setUp(self):
-        super(PoolAttributeTest, self).setUp()
+    CANCEL_FOR_TICKET = [
+        [
+            "DAOS-6971",
+            "test_method_name", "test_pool_attributes",
+            "test_method_name", "test_pool_attribute_asyn"
+        ],
+    ]
 
-        self.large_data_set = {}
+    @staticmethod
+    def create_data_set():
+        """Create the large attribute dictionary.
 
-        self.pool = TestPool(self.context, self.get_dmg_command())
-        self.pool.get_params(self)
-        self.pool.create()
-        self.pool.connect()
+        Returns:
+            dict: a large attribute dictionary
 
-    def create_data_set(self):
         """
-        To create the large attribute dictionary
+        data_set = {}
+        for index in range(1024):
```

Comment:
(pylint-unused-variable) Unused variable 'index'

Reply:
index variable will be renamed to "_"